gatsby-node.js

```diff
@@ -26,7 +26,7 @@ exports.onCreateDevServer = async ({ reporter }) => {
 Dynamic Pages
 ================================================================================
 */
-exports.createPages = async ({ graphql, actions, reporter }) => {
+createPages____UNUSED = async ({ graphql, actions, reporter }) => {
```

Comment:
I want to rely on the default page generation, not things from Contentful. Once we've got the site ready, we can iterate on the CMS side.